recipes/openassetio/all/test_package/conanfile.py

```diff
@@ -0,0 +1,74 @@
+import pathlib
+
+from conan import ConanFile
+from conan.tools.microsoft import is_msvc
+from conan.tools.build import can_run
+from conan.tools.cmake import cmake_layout, CMake, CMakeToolchain, CMakeDeps
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "CMakeDeps", "CMakeToolchain", "VirtualRunEnv"
+    test_type = "explicit"
+
+    def build_requirements(self):
+        self.tool_requires("cmake/3.25.1")
+
+    def requirements(self):
+        self.requires("openssl/1.1.1s")
```

Comment:
why do you need openssl here? In case it's a project dependency, it should be resolved automatically. I don't any of your tests import an OpenSSL header.

Reply:
It's an annoying dependency conflict between `cmake` and `cpython`, despite `cmake` being only a build requirement.  It appears to be a known issue with no solution yet e.g. https://github.com/conan-io/conan/issues/11287 and https://github.com/conan-io/conan/issues/11010.

I can add a code comment to clarify.

Reply:
Ah, code comment already added as part of 399e44b3

Reply:
#16775 should have solved this

```suggestion
        pass
```

Reply:
Updated in 748b5423